target/linux/ipq40xx/config-5.15

```diff
@@ -467,6 +467,7 @@ CONFIG_SPI_GPIO=y
 CONFIG_SPI_MASTER=y
 CONFIG_SPI_MEM=y
 CONFIG_SPI_QUP=y
+CONFIG_SPI_SPIDEV=y
```

Comment:
I would prefer if you add `kmod-spi-dev` to the `DEVICE_PACKAGES` instead. You already did that, so you can remove this. CONFIG_SPI_SPIDEV will then be build as module and included into the root file system.

Reply:
Sure, fixed